portfolio/src/main/webapp/script.js

```diff
@@ -124,6 +124,25 @@ function startSlideshow() {
  * and displays it on the page
  */
 function loadComments() {
+  const loginString = "/login";
+  fetch(loginString)
+    .then(response => response.json())
+    .then(json => {
+      const loggedIn = json["loggedin"];
+      const url = json["url"];
+      if (loggedIn) {
```

Comment:
To make it clearer that logged in users execute the JS following, use the format
```
if (!loggedIn) {
    <code for not logged in>
    return;
}
<code for logged in>
```